suites/upgrade/firefly-hammer-x/parallel/2-workload/rados_api.yaml

```diff
@@ -0,0 +1,10 @@
+workload:
+   sequential:
+   - workunit:
+       branch: firefly
+       clients:
+         client.0:
+         # Note: check if the test scrip used is good
+         - rados/test-upgrade-v9.0.1.sh
```

Comment:
this is a firefly -> hammer upgrade, so we should do whatever the firefly-x suite does here